blocks/article-body-block/chains/article-body/_children/pullquote.jsx

```diff
@@ -11,7 +11,7 @@ export default ({ element }) => {
       pullQuote.push(<p key={paragraph._id}>{ paragraph.content }</p>);
     }
   });
-  if (citation.type === 'text') {
+  if (citation.type === 'text' && citation.content !== (null || undefined || '')) {
```

Comment:
Not sure I totally follow this conditional. if the `citation.content` is `null`, we should not add the citation, right?

Reply:
```suggestion
  if (citation.type === 'text' && citation.content !== null  && citation.content !== undefined && citation.content !== '') {
```